src/main/java/dev/backend/UniTalk/model/Thread.java

```diff
@@ -0,0 +1,43 @@
+package dev.backend.UniTalk.model;
+
+import lombok.*;
+import javax.persistence.*;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "thread")
+@NoArgsConstructor
+@Getter @Setter
+public class Thread {
```

Comment:
Is it possible to test this class?

Reply:
to my current knowledge: not yet
first we need to create some basic controller